### PR TITLE
Issue 865

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://github.com/openstax/unified/issues/865

This should be merged after https://github.com/openstax/rex-web/pull/532

### CODE REVIEW

**Comment by TomWoodward** [comment]
@PiotrKozlowski can you merge or rebase this and resolve conflicts so we can review?

File: src/app/content/components/Page/PageContent.tsx [hunk 1]
```diff
@@ -60,6 +66,35 @@ export default styled(MainContent)`
...
+
+        &:after {
+          position: absolute;
+          z-index: -1;
+          content: "";
+          display: block;
+          width: calc(100% + 2rem);
+          height: calc(100% + 2rem);
+          top: -1rem;
+          left: -1rem;
```
Comment (TomWoodward) [inline src/app/content/components/Page/PageContent.tsx:80]: you could also set bottom and right to `-1` and you wouldn't need the calc in the width/height

**Review by TomWoodward — CHANGES_REQUESTED** [review]

File: src/app/content/components/Page/PageContent.tsx (hunk 1 above)
Comment (PiotrKozlowski) [reply src/app/content/components/Page/PageContent.tsx:79]: done

**Review by TomWoodward — APPROVED** [review]
needs conflicts merged, but otherwise looks good

**Pull request merged**